DQM/include/DQM/HgcrocPulseTruth.h

```diff
@@ -0,0 +1,39 @@
+#ifndef _DQM_HGCROCPULSETRUTH_H_
+#define _DQM_HGCROCPULSETRUTH_H_
+
+/*~~~~~~~~~~~~~~~*/
+/*   Framework   */
+/*~~~~~~~~~~~~~~~*/
+#include "Framework/Configure/Parameters.h"
+#include "Framework/Event.h"
+#include "Framework/EventProcessor.h"
+#include "TGraph.h"
+#include "Tools/AnalysisUtils.h"
+
+namespace dqm {
+
+class HgcrocPulseTruth : public framework::Analyzer {
```

Comment:
Can we call this class `HgcrocPulseTruthAnalyzer`? Or something that does not repeat the object name in Recon/include/Recon/Event/HgcrocPulseTruth.h